client/src/components/R&R/InteractiveStars.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
+import axios from 'axios';
+import styled from 'styled-components';
+
+const Star = ({ marked, starId }) => {
+  return (
+    <span value={starId} className="star" name="stars" role="button">
+      {marked ? '\u2605' : '\u2606'}
+    </span>
+  );
+};
+
+const StarRating = ({ value, handleInputChange }) => {
+  const [rating, setRating] = React.useState(parseInt(value) || 0);
+  const [selection, setSelection] = React.useState(0);
+
+  const multiFunc = event => {
+    setRating(event.target.getAttribute('value'));
+    handleInputChange(event);
+
+  }
+
+  const hoverOver = event => {
+    let val = 0;
+    if (event && event.target && event.target.getAttribute('value'))
```

Comment:
you can use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) to write this in one line:
`event?.target?.value`